events/dbpersist.go

```diff
@@ -63,6 +63,12 @@ func (p *DbPersistence) Persist(ctx context.Context, e *XRPCStreamEvent) error {
 
 	evt := e.RepoAppend
 
+	// TODO: hack hack hack
```

Comment:
Do you mean to ship this, even as a temporary thing?

I would pass through a "too large", or drop event, or remove all ops instead of this truncation.

Reply:
Yeah, the issue is that we currently don't have an official way of recovering from this. If we drop them then a consumer will get zero events instead of 'some', and for the purposes of the search engine work we're pushing on I would rather have some than none in the short term